examples/online_judge_programming/submit.py

```
@@ -0,0 +1,603 @@
+        if status_id > _RUNNING_STATUS:
+            # Done
+            print()
+            success = status_id == _ACCEPTED_STATUS
+            try:
+                root = fragment_fromstring(status["row_html"], create_parent=True)
+                cpu_time = root.xpath('.//*[@data-type="cpu"]')[0].text_content()
+                try:
+                    score = re.findall(
+                        r"\(([\d\.]+)\)", root.xpath('.//*[@data-type="status"]')[0].text_content()
+                    )[0]
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)